topology/University of Wisconsin/CHTC/CHTC-ITB.yaml

```
@@ -332,3 +332,30 @@ Resources:
     AllowedVOs:
       - GLOW
       - OSG
+
+  TIGER-OSG-BACKFILL-ITB:
+    Active: true
```

```suggestion
    Active: false
```
Not active yet. We'll change it when we set it up.